ramp-database/rampdb/model/submission.py

```diff
@@ -102,9 +187,16 @@ class Submission(Model):
     # later also ramp_id
     UniqueConstraint(event_team_id, name, name='ts_constraint')
 
-    def __init__(self, name, event_team):
+    # XXX: big change in the database
+    is_sandbox = Column(Boolean, default=False)
```

Comment:
Is this needed? 
Meaning: is it needed that this name is configurable? We can also hard-code that any submission named 'starting_kit' is a sandbox submission?